lib/web_ui/test/engine/semantics/text_field_test.dart

```diff
@@ -434,9 +365,332 @@ void testMain() {
         expect(appHostNode.activeElement, tester.getTextField(2).editableElement);
         expect(strategy.domElement, tester.getTextField(2).editableElement);
       }
+    });
+  }, skip: isIosSafari);
+
+  group('$SemanticsTextEditingStrategy in iOS', () {
+    late HybridTextEditing testTextEditing;
+    late SemanticsTextEditingStrategy strategy;
+
+    setUp(() {
+      testTextEditing = HybridTextEditing();
+      SemanticsTextEditingStrategy.ensureInitialized(testTextEditing);
+      strategy = SemanticsTextEditingStrategy.instance;
+      testTextEditing.debugTextEditingStrategyOverride = strategy;
+      testTextEditing.configuration = singlelineConfig;
+      debugBrowserEngineOverride = BrowserEngine.webkit;
+      debugOperatingSystemOverride = OperatingSystem.iOs;
+      semantics()
+        ..debugOverrideTimestampFunction(() => _testTime)
+        ..semanticsEnabled = true;
+    });
 
+    tearDown(() {
+      debugBrowserEngineOverride = null;
+      debugOperatingSystemOverride = null;
       semantics().semanticsEnabled = false;
     });
+
+    test('does not render a text field', () {
+      expect(appHostNode.querySelector('flt-semantics[role="textbox"]'), isNull);
+      createTextFieldSemanticsForIos(value: 'hello');
+      expect(appHostNode.querySelector('flt-semantics[role="textbox"]'), isNotNull);
+    });
+
+    test('tap detection works', () async {
+      final SemanticsActionLogger logger = SemanticsActionLogger();
+      createTextFieldSemanticsForIos(value: 'hello');
+
+      final DomElement textField = appHostNode
+          .querySelector('flt-semantics[role="textbox"]')!;
+
+      simulateTap(textField);
+      expect(await logger.idLog.first, 0);
+      expect(await logger.actionLog.first, ui.SemanticsAction.tap);
+    });
+
+      test('Syncs editing state from framework', () {
```

Comment:
There's an extra indent compared to sigling tests.

Reply:
Done!